.github/workflows/run_tests.yml

```diff
@@ -235,12 +235,11 @@ jobs:
       - name: Install go-swagger
         id: install-go-swagger
         run: |
-          sudo apt-get update
-          sudo apt-get install -y apt-transport-https gnupg curl
-          curl -1sLf 'https://dl.cloudsmith.io/public/go-swagger/go-swagger/gpg.2F8CB673971B5C9E.key' | sudo apt-key add -
-          curl -1sLf 'https://dl.cloudsmith.io/public/go-swagger/go-swagger/config.deb.txt?distro=debian&codename=any-version' | sudo tee /etc/apt/sources.list.d/go-swagger-go-swagger.list
-          sudo apt-get update
-          sudo apt-get install -y swagger
+          dir=$(mktemp -d)
+          git clone https://github.com/go-swagger/go-swagger "$dir" 
```

Comment:
As mentioned on Slack, I don't think we need to git clone and build from source. The wget method in https://github.com/CMSgov/easi-app/pull/1644/commits/181023013bb0dee9d9439d9ff41b32387388f6c9 was nearly fine, it just needed the right command line flag and using v0.28.0 instead of v0.29.0.

Reply:
Yeah - I'm good with this method as well. Though - if what we have works, I'd also be fine with this method (but either way I think we should update to v0.28.0